projects/cashmere/src/lib/form-field/hc-form-field.component.html

```diff
@@ -1,4 +1,9 @@
-<div [class.hc-form-field-wrapper-inline]="inline" [class.hc-form-field-wrapper]="!inline" [class.hc-form-field-no-label]="!hasLabel">
+<div
+    [class.hc-form-field-wrapper-inline]="inline"
+    [class.hc-form-field-wrapper]="!inline"
+    [class.hc-form-field-no-label]="!hasLabel"
+    [class.hc-form-field-padding]="!tight"
```

Comment:
Hey @jonest - line 5 here is actually the one to look at (the others were already there before).  Basically what I did is pull all the padding or margin values in components out into their own separate classes.  And a developer can now set `tight="true"` on any of these components to omit the default padding - ie. omit that padding class.  So what you've got here is binding that padding class to the inverse value of of the tight param.  In other words, if tight is true, then don't include the padding class.

Reply:
OK got it, so it's like a global binding to change it everywhere (a step toward a "tight" UI across the board then?)

Reply:
@jonest - that's mostly correct, I've added a `tight` parameter to every component that has default padding or margin.  But it's not global in the sense that you set a value once and it applies it everywhere.  It's still applied on an instance by instance basis.  So for example I could have `<table hc-table tight="true">` for one table that is using the compressed style, but another `<table hc-table>` that still has the default padding.